modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc

```diff
@@ -68,6 +68,15 @@ For the schedule of upcoming releases, see <link that has the dates>.
 [NOTE]
 You must have an Enterprise support account to view the release schedule.
 
+You might need to update your applications manually, for example:
+
+* The automatic update schedule doesn't work in your organization.
+* Your application doesn't support zero-downtime updates. 
```

Comment:
Can we change the above 3 lines to something like this:

If your application does not support zero-downtime updates or requires a manual update for any reason, the date patch released in the first week of the month can be taken manually by following the instructions in " Update the Runtime Version Manually". If you have several applications to update at a time, the bulk api can be used for your update <link https://help.mulesoft.com/s/article/How-to-Perform-a-Bulk-CloudHub-Application-Update-to-the-Latest-Runtime-Releases-Using-a-Unix-Shell-Script>